### ISSUE TITLE
Android app application

### ISSUE THREAD

**Issue by Pierochr** [jimmyH/BeSIM-GUI#2]
Hi, could be possible in your opinion to create a simple android app for besmart in order to substitute the original one?

**Comment by jimmyH** [comment]
Yes, it is possible (but you would still need a server for BeSim).

Note: If you use "Add to Home screen" in chrome you can get make this web GUI look like an app.

**Comment by Pierochr** [comment]
Hi thanks.

Could you explain how i can launch this on chrome first? sorry but i don't understand

**Comment by jimmyH** [comment]
You need to run:
- BeSIM server (https://github.com/jimmyH/BeSIM) which is what the BeSmart device in your home communicates with.
- This BeSIM-GUI, which hosts the webpage your browser connects to.

**Comment by Pierochr** [comment]
Thanks, but it's too complicated for my knowledge. Di you have a simple guide telling me step by step how I should to procede?or a android application should be more easier...

**Comment by jimmyH** [comment]
You cannot just use a standalone android application. The BeSmart device is designed to communicate with a server.

If you want simple, you need to use the BeSmart/RiCloud android applications which communicate with Riello's servers.
